From 59cd43f2510971e9590485f0a04c37a7aec9ead5 Mon Sep 17 00:00:00 2001
From: Mord3rca <morderca@morderca.net>
Date: Sun, 1 Oct 2023 17:05:58 +0200
Subject: [PATCH] Fixing missing inclusion in flags.h

---
 include/pistache/flags.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/pistache/flags.h b/include/pistache/flags.h
index 9be2b32..6adcc74 100644
--- a/include/pistache/flags.h
+++ b/include/pistache/flags.h
@@ -13,6 +13,7 @@
 #pragma once
 
 #include <climits>
+#include <cstdint>
 #include <iostream>
 #include <type_traits>
 
-- 
2.41.0

